generate-map.py

```diff
@@ -3,38 +3,84 @@
 import sqlite3
 import json
 from geopy.geocoders import Nominatim
+import os
 
 
 def getUserInformation():
-    geolocator = Nominatim(timeout=10, user_agent='Opencast map generator')
+
+    locations = []
 
     cur = sqlite3.connect('user.db').cursor()
     cur.execute('select distinct country, city, organization from user')
     for country, city, organization in cur.fetchall():
-        location = geolocator.geocode({'country': country, 'city': city},
-                                      addressdetails=True) \
-                   or geolocator.geocode('%s, %s' % (country, city),
-                                         addressdetails=True)
-        print((country, city), '->', location)
-        if location:
-            yield (location.latitude, location.longitude, organization)
+        if organization != "None":
+            locations.append(compareCache(country, city, organization))
+    return locations
+
+
+def getGeoCode(country, city, organization, dataList):
+
+    geolocator = Nominatim(timeout=10, user_agent='Opencast map generator')
```

Comment:
We do not need to initialize this every time. Maybe move this to a module level?